.github/workflows/checks.yml

```
@@ -12,23 +12,16 @@ jobs:
     strategy:
       matrix:
         ruby: ['2.4', '2.5', '2.6', '2.7', '3.0']
-        rubocop: ['0.63.1', '0.84.0', '0.85.0', '0.86.0', '0.87.0', '< 1.0']
-        include:
-          - ruby: '2.3'
-            rubocop: '0.63.1'
-          - ruby: '2.3'
-            rubocop: '0.81.0'
+        rubocop: ['0.63.1', '0.81.0', '0.84.0', '0.85.0', '0.86.0', '0.87.0', '1.0.0', '1.x']
```

```suggestion
        rubocop: ['0.63.1', '0.81.0', '0.84.0', '0.85.0', '0.86.0', '0.87.0', '< 1.0', '1.0.0', '< 2.0']
```
So that all versions are covered.

---

OK. Changed.